fm.c

```diff
@@ -79,21 +79,19 @@ void FM_Prescaler(fm_prescaler_t *chip)
 
 void FM_HandleIO(fm_t *chip)
 {
-    int write_data = chip->input.cs && chip->input.wr && (chip->input.address & 1) == 1 && !chip->input.ic;
-    int write_addr = (chip->input.cs && chip->input.wr && (chip->input.address & 1) == 0) || chip->input.ic;
-    int read_enable = chip->input.cs && chip->input.rd && !chip->input.ic;
-    int io_dir = chip->input.cs && chip->input.rd && !chip->input.ic;
-    int data_enable = !io_dir && !chip->input.ic;
+    // Unused - commented out until information is provided.
+    //const int io_dir = chip->input.cs && chip->input.rd && !chip->input.ic;
+    //const int data_enable = !io_dir && !chip->input.ic;
 
     if (chip->input.cs && chip->input.wr)
     {
         chip->data_latch = chip->input.data;
         chip->bank_latch = (chip->input.address >> 1) & 1;
     }
 
-    if (write_addr)

```

Comment:
no speed gain.

Reply:
This isn't for a gain of speed; it cleans up stack memory as these values could be constant but for the most part are only used once while defined.